### PR TITLE
Re-organizing some test stuff.

### PR DESCRIPTION
**Pull request by jonparrott**
* Bigquery and storage tests extend from common tests.CloudBaseTest.
* All test resources now live in tests/resources, accessible via self.resources inside of tests.
* Updated tox to allow testing a single file for dev purposes.

### CODE REVIEW

**Comment by jonparrott** [comment]
@tmatsuo PTAL

File: tox.ini [hunk 1]
```diff
@@ -22,5 +22,5 @@ commands =
     cover: --cover-tests --cover-branches --cover-min-percentage=70 \
     cover: --cover-inclusive --cover-erase
     cover: coveralls
-    nosetest: nosetests --with-gae
+    nosetest: nosetests --with-gae []
```
Comment (tmatsuo) [inline tox.ini]: What is this [] for?

Comment (jonparrott) [inline tox.ini]: Passes arguments given to tox into nosetests

**Comment by tmatsuo** [comment]
Because the travis is using secrets, we can not run the tests for pull requests from other repo, even it's coming from googlers. Maybe you can create a pull request from a branch in the same repo next time, so that travis will be able to run the test.

**Comment by tmatsuo** [comment]
LGTM

**Pull request merged**